docs/assets/js/_partials/chart-recommended-git-versions.js

```diff
@@ -9,7 +9,9 @@ class RecommendedGitVersionsChart extends AreaChart
                 vulnerable: 'red',
                 outdated: 'yellow',
                 recommended: 'green',
+                unknown: 'grey',
```

Comment:
Some libGit clients report libGit versions numbers that can easily be confused with real Git versions numbers. Would they count here?